docs/quickstart.md

```diff
@@ -7,7 +7,7 @@ This guide covers how you can quickly get started using Draft
 The following prerequisites are required for successful use of Draft.
 
 1. A Kubernetes cluster and the `kubectl` CLI tool
-2. Installation of Helm on your Kubernetes cluster
+2. Installation of [Helm](https://github.com/Kubernetes/helm) on your Kubernetes cluster
```

Comment:
should kubernetes be lowercase?

Reply:
I don't actually know. Let me look around!

Reply:
It looks like in other places in our docs and comments, it is capitalized.

Reply:
I'll leave as is to preserve consistency but open to changing in the future if necessary!